pkgs/applications/networking/cluster/kube-capacity/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kube-capacity";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "robscott";
+    repo = "${pname}";
+    sha256 = "1h5afm8njaw8rbpzr7yha6036g5d1payjxyn9wig50aq7d9s7r1s";
+  };
+
+  vendorSha256 = "1z5k2l5q043pyxb494k951iv1ika5wifczxprwxica212s3bv8x8";
+
+  doCheck = false;
```

Comment:
Please add a comment why you disabled tests.

Reply:
Added tests back for latest release.